apps/protailwind/src/templates/workshop-template.tsx

```diff
@@ -10,21 +10,41 @@ import {isBrowser} from 'utils/is-browser'
 import {track} from '@skillrecordings/skill-lesson/utils/analytics'
 import first from 'lodash/first'
 import * as Accordion from '@radix-ui/react-accordion'
-import {CheckIcon, ChevronDownIcon} from '@heroicons/react/solid'
+import {
+  CheckIcon,
+  ChevronDownIcon,
+  LockClosedIcon,
+  PlayIcon,
+} from '@heroicons/react/solid'
 import {trpc} from 'utils/trpc'
 import {find, isArray} from 'lodash'
 import {LessonResource} from '@skillrecordings/skill-lesson/schemas/lesson-resource'
 import PortableTextComponents from '../video/portable-text'
+import {Pricing} from 'path-to-purchase-react/pricing'
+import {
+  CommerceProps,
+  SanityProduct,
+} from '@skillrecordings/commerce-server/dist/@types'
+import {useCoupon} from 'path-to-purchase-react/use-coupon'
+import {PriceCheckProvider} from 'path-to-purchase-react/pricing-check-context'
 
 const WorkshopTemplate: React.FC<{
   workshop: SanityDocument
-}> = ({workshop}) => {
-  const {title, body, ogImage, image, description} = workshop
+  commerceProps?: CommerceProps
```

Comment:
these are optional and loaded async via trpc, probably don't need to be passed in? could just load them here.